app/src/main/java/com/lastaoutdoor/lasta/models/user/Converters.kt

```diff
@@ -0,0 +1,96 @@
+package com.lastaoutdoor.lasta.models.user
+
+import androidx.room.TypeConverter
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+import com.lastaoutdoor.lasta.models.activity.ActivityType
+import com.lastaoutdoor.lasta.models.activity.ClimbingStyle
+import com.lastaoutdoor.lasta.models.activity.Difficulty
+import com.lastaoutdoor.lasta.models.activity.Rating
+
+class Converters {
+  @TypeConverter
+  fun fromClimbingStyle(c: ClimbingStyle): String {
+    return c.toString()
+  }
+
+  @TypeConverter
+  fun toClimbingSyle(s: String): ClimbingStyle {
+    return if (s == "Outdoor") {
+      ClimbingStyle.OUTDOOR
+    } else {
```

Comment:
Maybe be careful with this else if in the future we add a third climbing style or change the naming